### DISCUSSION TITLE
WIndows App Sdk WinUI3 and LiveChart2

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#9193] [Q&A] [answered]
https://platform.uno/blog/livecharts-announces-support-for-uno-platform/

![image](https://user-images.githubusercontent.com/95089394/176992550-f2faa0a9-7027-455b-8ba2-155d373bafd3.png)
![LiveChart2Problem](https://user-images.githubusercontent.com/95089394/176992568-177e54c6-adf4-4bec-8b03-d5953b57f1da.png)
![image](https://user-images.githubusercontent.com/95089394/176992848-6749b1a6-6737-4f4f-8b5b-f1393d4dbe25.png)

I also tried on a new empty DummieProject (Standardtemplate Net6) windows app sdk template version 4.5
After adding the LiveChart nuget for Uno, al heads fail..

**https://platform.uno/blog/livecharts-announces-support-for-uno-platform/**

This blog is UWP/WinUI2 Mobile net6. The Standard UNO template net6 ships with Windows App Sdk/WinUI3
the Question is **does LiveCharts2 already works with the new Template? and if not when and where can i follow the progress?**

**Comment by Daantje007** [comment, marked as answer]
This is such a beautiful library which can not be missed in Uno.

**Reply by francoistanguay** [reply]
See https://github.com/beto-rodriguez/LiveCharts2/issues/492

**Reply by beto-rodriguez** [reply]
@Daantje007 it will be supported, I will update the package to WinUI soon.

**Reply by wuweisin** [reply]
@beto-rodriguez ,I'm also looking forward to your work...
 Thanks for your contribution , this will be very useful !
